extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -1155,6 +1241,35 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 		});
 	}
 
+	/**
+	 * Creates input box for filegroup name (declarative)
+	 * @param table declarative table component
+	 * @param filegroupType filegroup type
+	 * @returns Input component
+	 */
+	private getFilegroupNameInputDeclarative(table: azdata.DeclarativeTableComponent, filegroupType: FileGroupType): azdata.InputBoxComponent {
```

Comment:
It looks like most of the code here is the same as in the function above in `getFilegroupNameInput()`. Will the above function eventually be removed after the other tables are converted? If so, then not worth spending time refactoring for more code reuse.

Reply:
Yes it will be removed after all the other tables are converted